src/main/java/javaserver/handlers/FileHandler.java

```diff
@@ -23,24 +24,17 @@ public FileHandler(String fileUri, ResourceUtility resourceUtility) {
 
     public Response handleRequest(Request request) {
 
-        String method = request.getMethod();
-        if (!method.equals("GET")) {
+        if (!isGetRequest(request)) {
             response.setStatus(Status.MethodNotAllowed);
             return response;
         }
 
-        if (isPartialRequest(request)) {
-            response = new PartialHandler(fileUri, resourceUtility).handleRequest(request);
-            return response;
-        }
-
-        if (isImageRequest(request)) {
-            response = new ImageHandler(fileUri, resourceUtility).handleRequest(request);
-            return response;
+        Handler fileHandler = getTypeOfFileHandler(request);
```

Comment:
Oh i see, so still within the FileHandler. Ok I'll think about those two options and see which one makes the most sense.